File: MODERN-ART-ASSOCIATION-OF-KAOHSIUNG/JS/Artisticworks.js
$(document).ready(function () {
    $.ajax({
        url: '/Artisticworks/GetData',
        type: 'GET',

        dataType: 'json',
        success: function (data) {
            //data[0].Artisticworks_Date = "123";
            console.log(data);
        },
        error: function () {
            console.log("無法取得db");
        }
    });
    //$.ajax({
    //    url: '/Artisticworks/CreateArtisticwork',
    //    type: 'POST',
    //    data: {
    //        Artisticworks_Name: 'test22',
    //        Artisticworks_Date: '2015/08/23',
    //        Artisticworks_Picture: '暫無資料222',
    //        Member_ID: 7
    //    },
    //    dataType: 'json',
    //    success: function (data) {
    //        console.log("test");
    //    },
    //    error: function () {
    //        console.log("無法取得db");
    //    }
    //});
    //$.ajax({
    //    url: '/Artisticworks/UpdateArtisticwork',
    //    type: 'POST',
    //    data: {
    //        Artisticworks_Name: 'test111',
    //        Artisticworks_Date: '2015/08/12',
    //        Artisticworks_Picture: '暫無資料12345',
    //        Member_ID: 14,
    //        Artisticworks_ID: 46
    //    },
    //    dataType: 'json',
    //    success: function (data) {
    //        console.log("test");
    //    },
    //    error: function () {
    //        console.log("無法取得db");
    //    }
    //});
    //$.ajax({
    //    url: '/Artisticworks/DeleteArtisticwork',
    //    type: 'POST',
    //    data: {
    //        Artisticworks_ID: 45,
    //    },
    //    dataType: 'json',
    //    success: function (data) {
    //        console.log("test");
    //    },
    //    error: function () {
    //        console.log("無法取得db");
    //    }
    //});
});

File: MODERN-ART-ASSOCIATION-OF-KAOHSIUNG/Models/TableModel/Artisticworks.cs
namespace MODERN_ART_ASSOCIATION_OF_KAOHSIUNG.Models.TableModel
{
    public class Artisticworks
    {
        /// <summary>
        /// 資料表名稱
        /// </summary>
        public string TABLE_NAME { get { return "Artisticworks"; } }
        
        /// <summary>
        /// 作品日期
        /// </summary>
        public string Artisticworks_Date { get; set; }

        /// <summary>
        /// 作品編號
        /// </summary>
        public string Artisticworks_ID { get; set; }

        /// <summary>
        /// 作品名稱
        /// </summary>
        public string Artisticworks_Name { get; set; }

        /// <summary>
        /// 作品
        /// </summary>
        public string Artisticworks_Picture { get; set; }

        /// <summary>
        /// 會員編號
        /// </summary>
        public string Member_ID { get; set; }

        public void PK_Artisticworks(string Artisticworks_ID)
        {

        }
    }
}